public/provisioning.php

```diff
@@ -102,10 +102,10 @@
 
     $data = getDataFromFilename($filename,$this->logger);
     $this->logger->debug(print_r($data,true));
-    if (array_key_exists('scope_id',$data) and !empty($data['scope_id'])) {
-        $id = $data['scope_id'];
+    if (array_key_exists('scopeid',$data) and !empty($data['scopeid'])) {
```

Comment:
How is it related to Yealink templates? It's better to split it to a different PR...